### PR TITLE
Pull request for The next web clone

### PR DESCRIPTION
**Pull request by berhanu-tarekegn**
**The next web clone**
- This is the first **PR** for the whole clone
- has done through 
**CSS** 
**HTML5**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,13 +51,30 @@
...
+                <div class="section bg2">
+                    <h2 >
+                        3 tech stocks that surged in price this year — but didn’t get much attention
+                    </h2>
+                </div>
+                <div class="section bg3">
+                    <h2 >
+                        3 tech stocks that surged in price this year — but didn’t get much attention
+                    </h2>
+                </div>
```
Comment (addod19) [inline index.html]: Let us improve semantics by using ```<article>``` instead of ```<div>```. Each of the ```<div>``` should be ```<article>```

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 2]
```diff
@@ -116,371 +140,636 @@ <h4>Dangerous AI algorithms and how to recognize them</h4>
                 </li>
             </ul>
         </section>
+        <section>
+            <h2 class="latest-heading">
+                Latest funding rounds on Index.co New Window >
+            </h2>
+            <ul class="listing">
+                <li class="latest-news fund">
```
Comment (addod19) [inline index.html:153]: Each of the ```<li>``` should be ```<article>``` so we can improve on HTML semantics

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @umairarshadbutt and @berhanu-tarekegn, great job with this project :+1: README is very professional. :100:  Let us get our project approved by addressing the comments below: :smile: 

- PR must be a project description. You can go back to the project requirements and you will find a suitable description of the project
- The description of the README could also be improved. You can think of the description, as what Microverse wants you to learn in building the project. And you will find it in the project requirements
- ``` Medium``` The TNW logo is not appearing on the leftmost part of the page
md
- ``` Large ``` We are missing the TNW logo at the top left corner of the row
tnw
- ``` Footer ``` We are missing TNW logo
ttt
- Let improve semantics by replacing all the articles with the <article> tags 
- Let us implement the DRY rule in our CSS file

**NB:** I realized 3 of you participated in the project. In that case, you could explain why it happened

Kindly implement these changes and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: index.html (hunk 2 above)
Comment (umairarshadbutt) [reply index.html:153]: The Original  live website is using listing for this section

File: index.html (hunk 1 above)
Comment (umairarshadbutt) [reply index.html]: done

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hello everyone  :hand: 

### Good job, you did great job with the project :+1: 

### Unfortunately, there are still some changes to get the project approved:
- **You did not implement any changes required by the previous TSE, please go back and implement all of them**
- Fix the validators issues:
Screenshot from 2020-06-23 00-26-44
- You have the whole first section missing in your project:
original
Screenshot from 2020-06-23 00-27-53
your project:

Screenshot from 2020-06-23 00-29-21

Please add it.
- Don't forget to provide an explanation for 3 person working on the same project.

### Once done with the changes, please ask for another code review.

**Comment by umairarshadbutt** [comment]
- After Refreshing page, you got this section
- My partner Used two git accounts due to his using two accounts we are three at the project including me.

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey Team :wave:
-
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
- For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.

### Main Page

|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/berhanu-tarekegn/the-next-web-clone/develop/index.html)|
|:---:|:---:|
|source-page|clone|

- Increase the height of the top three articles of your clone to match that of the source page.
- Add the TWN logo at the top left corner of the main article ([link to page](https://thenextweb.com/)).
##
### Latest News Section
|[Source Page](https://thenextweb.com/)|[Clone page](https://raw.githack.com/berhanu-tarekegn/the-next-web-clone/develop/index.html)|
|:---:|:---:|
|latest-news|latest-news-clone|

- This section should align only 4 articles per row ([link to page](https://thenextweb.com/)). Let's work on that.
- Make sure this and the latest funding section have properly been centred.
##
### Latest Funding Section

|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/berhanu-tarekegn/the-next-web-clone/develop/index.html)|
|:---:|:---:|
|latest-funding|latest-funding-clone|

- Centre this section.
##

### Categories (Apps, Gear, Tech, etc) section

|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/berhanu-tarekegn/the-next-web-clone/develop/index.html)|
|:---:|:---:|
|app-gear|app-gear-clone|

- It should align 3 category boxes per row ([link to page](https://thenextweb.com/))
##

### Latest Deals Section
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/berhanu-tarekegn/the-next-web-clone/develop/index.html)|
|:---:|:---:|
|latest-deals|latest-deals-clone|

- The latest deals should as well align four articles per row.
##

- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by trekab — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
Hi @team, I'd like to start by commending you for the effort you are putting in to meet this project's requirements. There are a few changes you need to make to complete. These are:

- [x] Please make your navigation links actual links and label them more appropriately. This will make your clone appear more like the original site.
- [x] Your navigation bar is broken on medium-size screens.
- [x] For medium and small screens, the TNW logo should disappear from the `Top 3 articles section`.
- [x] For the `Top 3 articles section` your largest article is missing its bottom text.

Screenshot

- [ ] At 862px - 863px, your `Latest news`, `Latest funding`, and `Latest deals` sections seem to be responding incorrectly. Try to refine your break-points. Your CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/breakpoints_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/)). 
  - No small breakpoint as it is mobile-first and the base styles are for the mobile screen. ([Screenshot on mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
  - medium: min-width around 768px
  - large: min-width 1024px

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hi everyone   :hand: 

### Great project, godd work, you implemented a lot of required changes, but there is some changes to be made:
- Please, always make sure that you page is not messed up, the navigation bar need improvement:
your project:
Screenshot from 2020-06-23 11-48-59
original:
Screenshot from 2020-06-23 11-54-08
- There is some overlapping text in small size:
Screenshot from 2020-06-23 12-00-43
Screenshot from 2020-06-23 12-01-59
As rule of thumb that you need to follow is that your final product need to be looking good, nothing messed up, so before you submit again for code review, check your project using the developer tool, check different screen size and fix any issue.

### Once done with the changes, ask for another code review.

**Review by trekab — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
Hi @team, much as you have endeavored to work on some of the previous feedback from the TSEs, it appears that you have neglected some other important feedback. Please note that we are here to help you learn and I encourage you to reach out to any of the TSEs if you feel that the feedback given is unclear. On that note, among other things, I still find a problem with the responsiveness of your clone. Given that the goal of this project is to help us appreciate and master working with responsive design, I'd like to point out the following:
- [ ] At the __same__ point that the TNW logo disappears from the `Top 3 articles section`, the `latest news section`, `Latest Funding section`, and the `Latest deals sections` align 3 articles per row while the `Categories (Apps, Gear, Tech, etc) section` aligns 2 category boxes per row ([link to page](https://thenextweb.com/)). Your clone behaves differently from this. I suggest that you look at my last feedback about break-points. This seems like an easier route for you to fix this issue.

__Kindly make the necessary changes and submit another code review request.__
_Again, feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @umairarshadbutt @BerhanuTarekegn** 
Good job with your  progress on this project :muscle:  
There are still some improvements to do before you can go ahead:
- your page should show the TNW logo when big screen size:



- In medium-size screen you page look like this, check the original page for the same size:



- For small screen size let compare your page and the original. Yours:



and the source page:



Besides you page still has some other issues:
- Latest news section should be two columns in medium-size screens
- lastest funding should  be two columns in medium size screen
- latest deals section should be three columns in medium size screen
- check how the footer changes in the original page in small screen size

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Status: Approved ✔

## You have done all the changes requested by the previous reviewers. 

### However, I highly recommend that you take note of these [OPTIONAL] changes below.

#### [OPTIONAL]
- Add the red gradient background to the top article sections.
- Take note of this ->
Annotation 2020-06-23 230635
There shouldn't be any extra space.

Original
image

Consider applying these changes before merging.

Happy coding 👨‍💻

**Pull request merged**